anchore_engine/db/entities/upgrade.py

```diff
@@ -1865,9 +1865,119 @@ def remove_policy_engine_sizes():
             raise ex
 
 
+def upgrade_oauth_client_014():
+    """
+    Changes required in db model for moving from authlib 0.12.3 -> 0.15.2
+    New column in db as defined in the Authlib oauth2 client mixin type
+
+    :return:
+    """
+
+    # TODO: should probably just drop and re-create this table for upgrade
+    # Would only affect sessions/tokens that can be reset after a system upgrade anyway.
+
+    engine = anchore_engine.db.entities.common.get_engine()
+
+    new_columns = [
+        {
+            "table_name": "oauth2_clients",
+            "columns": [
+                Column(
+                    "client_metadata",
+                    Text,
+                ),
+                Column(
+                    "client_id_issued_at",
+                    Integer,
+                ),
+                Column(
+                    "client_secret_expires_at",
+                    Integer,
+                ),
+            ],
+        }
+    ]
+
+    log.err("creating new table columns")
+    for table in new_columns:
+        for column in table["columns"]:
+            log.err(
```

Comment:
why is this an error level log statement?

Reply:
the only output during upgrade is stderr since it runs before webstack/logger bootstrap